pinot-server/src/main/java/org/apache/pinot/server/starter/helix/BaseServerStarter.java

```diff
@@ -664,7 +680,8 @@ public void start()
     } catch (Exception e) {
       LOGGER.error("Failed to register DefaultClusterConfigChangeHandler as the Helix ClusterConfigChangeListener", e);
     }
-    _clusterConfigChangeHandler.registerClusterConfigChangeListener(_segmentPreprocessThrottler);
+    _clusterConfigChangeHandler.registerClusterConfigChangeListener(_segmentAllIndexPreprocessThrottler);
+    _clusterConfigChangeHandler.registerClusterConfigChangeListener(_segmentStarTreePreprocessThrottler);
```

Comment:
Suggest making the top level `_segmentPreprocessThrottler` a listener and only register that. We can use the top level throttler to delegate the cluster change callback

Reply:
done, can you please take a look, thanks!